baus/datasources.py

```diff
@@ -612,9 +621,8 @@ def reprocess_dev_projects(df):
 
 # shared between demolish and build tables below
 def get_dev_projects_table(parcels, run_setup):
-    df = pd.read_csv(os.path.join(orca.get_injectable("inputs_dir"), 
-                     "basis_inputs/parcels_buildings_agents/development_pipeline_with_basis_buildings_nov2023.csv"), 
-                     dtype={'PARCEL_ID': np.int64, 'geom_id':   np.int64})
+    df = pd.read_csv(os.path.join(orca.get_injectable("inputs_dir"), "basis/parcels_buildings_agents",
+                     run_setup["development_pipeline_file"]), dtype={'PARCEL_ID': np.int64, 'geom_id':   np.int64})
```

Comment:
We could use something like run_setup.get('development_pipeline_file', '2021_0309_1939_development_projects.csv'), to offer a default in case the user hasn't speficied a pipeline file key. This would allow more minimal specs. And of course avoid a crash if the key is not defined. The downside is _not_ having the key in there may lead to folks inadvertently thinking they are running with something else; so less explicit.

Reply:
We used to do something like that with accessibility files and it made it tricky to see what file was actually used when there was optionality in the code. But I see your point too since you could look at the inputs folder for the run and the default file should be here.

I think I prefer this version in that the `run_setup.yaml` output will always show what file version was used and the template should be copied from the latest version of the file, like the one stored in the repo.